roles/caliopen/tasks/main.yaml

```diff
@@ -81,6 +78,11 @@
   args:
     chdir: "{{ devtools_dir }}"
 
+- name: setup notif ttls
```

Comment:
it's no more needed to setup distcintly notif_ttls, it's done by the `setup` cli command